submit-web/src/components/Shared/Table/utils.ts

```diff
@@ -1,4 +1,5 @@
-import { downloadObject } from "@/hooks/api/useObjectStorage";
+import { downloadObject, S3_FOLDER } from "@/hooks/api/useObjectStorage";
+import { camelCase } from "lodash";
```

Comment:
I didn't love that I'm using lodash, but we are already using this camelCase for the url. So no new lodash code was added. I just moved it to the utils